src/preferences/dialog/dlgprefautodjdlg.ui

```diff
@@ -253,6 +253,59 @@
     </widget>
    </item>
 
+<item>
+    <widget class="QGroupBox" name="CrossfaderBehaviour">
+      <property name="title">
+       <string>Crossfader Behaviour</string>
+      </property>
+      <property name="alignment">
+       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
+      </property>
+      <layout class="QGridLayout" name="CrossfaderBehaviourLayout">
+
+        <item row="0" column="0">
+          <widget class="QLabel" name="XFaderCenterResetStateLabel">
+            <property name="sizePolicy">
+              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+              </sizepolicy>
+            </property>            
+            <property name="text">
+              <string>Reset the Crossfader back to center after disabling AutoDJ</string>
+            </property>
+            <property name="buddy">
+              <cstring>XFaderCenterResetStateCheckBox</cstring>
+            </property>
+          </widget>
+        </item>
+
+        <item row="0" column="1">
+          <widget class="QCheckBox" name="XFaderCenterResetStateCheckBox">
+            <property name="toolTip">
+              <string>Reset the Crossfader back to center after disabling AutoDJ</string>
+            </property>
+          </widget>
+        </item>
```

Comment:
I think we definitely have to add a note here explaining the consequences (main gain drop), see #10683